### PR TITLE
Function to validate interaction

### PR DESCRIPTION
**Pull request by szynwelski**

### CODE REVIEW

File: src/utils/smartweave/utils.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"github.com/warp-contracts/syncer/src/utils/arweave"
+	"github.com/warp-contracts/syncer/src/utils/tool"
+)
+
+func ValidateInteraction(tx *arweave.Transaction) error {
+	if tx.Format < 2 {
+		return errors.New("interaction should not have a format smaller than 2")
+	}
+
+	tags := tx.GetTagMap(TagAppName, TagContractTxId, TagInput)
```
Comment (janekolszak) [inline src/utils/smartweave/utils.go:16]: this assumes there's only one tag of each type, there can be many

Comment (janekolszak) [reply src/utils/smartweave/utils.go:16]: For example there can be many App-Name tags

Comment (janekolszak) [reply src/utils/smartweave/utils.go:16]: Creating a map seems to be an overkill, one iteration through tags should be enough.

Comment (szynwelski) [reply src/utils/smartweave/utils.go]: fixed

**Pull request merged**